### ISSUE TITLE
Pagination loading

### ISSUE THREAD

**Issue by wanyunr** [prasathmani/tinyfilemanager#1152]
When the number of files in the directory is too large, the loading will be very easy to get stuck, if it can be loaded in pages, I think it will be much better

**Comment by parera10** [comment]
Yes, could be so cool to paginate in bunches of files instead of trying to get info from all the files in the folder. I have a folder with +70k files. An 
`ls -l`
in the server only take a few seconds but >5min in the browser and lot of times it stucks.
